### ISSUE TITLE
OMOptim does not communicate with OMC

### ISSUE THREAD

**Issue by anotheruserofgithub** [OpenModelica/OpenModelica#9259]
### Description

For fun I wanted to try using OMOptim, so I compiled it from source successfully but encountered the same problem as reported [here](https://trac.openmodelica.org/OpenModelica/ticket/4487).

Apparently this is simply a filename issue (on Linux at least).

Content of `/tmp/OpenModelica_user/OMOptim/OMLog.txt`:
```
13:38:24	OMC: attempt to start
13:38:24	OMC: running command: /home/user/OpenModelica/build/bin/omc +c=OMOptim10565ea91b8685003e165857630420426000000026170075133824204 +d=interactiveCorba +corbaObjectReferenceFilePath=/tmp/OpenModelica_user/OMOptim/
13:38:45	OMC: unable to find OMC server, Object reference file /tmp/OpenModelica_user/OMOptim//openmodelica.objid.OMOptim10565ea91b8685003e165857630420426000000026170075133824204 not created.
13:38:45	OMC: getClassNames()
13:38:45	OMC not started. Please start it again (menu OMC->restart)
```

Content of `/tmp/OpenModelica_user/OMOptim/openmodelica.user.omc.output.OMOptim`:
```
The flag -d=interactiveCorba is depreciated. Please use --interactive=corba instead.
Created server.
Dumped Corba IOR in file: /tmp/OpenModelica_user/OMOptim//openmodelica.user.objid.OMOptim10565ea91b8685003e165857630420426000000026170075133824204
Started the Corba ORB thread with id: 140058396800768
```

Content of `/tmp/OpenModelica_user/OMOptim/openmodelica.user.objid.OMOptim10565ea91b8685003e165857630420426000000026170075133824204`:
```
IOR:010000001900000049444c3a4f6d63436f6d6d756e69636174696f6e3a312e30000000000100000000000000a4000000010102000a0000003132372e302e302e3100078a51000000ff4f4d43504f41feb0dddb6201003808004f4d4f7074696d3130353635656139316238363835303033653136353835373633303432303432363030303030303032363137303037353133333832343230340000000200000000000000080000000100000000545441010000001c00000001000000010001000100000001000105090101000100000009010100
```

It seems that on the one hand (OMOptim), the generated file contains the user name (`openmodelica.user.objid.OMOptim*`), whilst on the other hand (OMC), the expected file does not contain the user name (`openmodelica.objid.OMOptim*`).

### Steps to Reproduce

Open OMOptim and check the files generated in `/tmp/OpenModelica_user/OMOptim/`.

### Expected Behavior

OMC should be able to start when launching OMOptim and CORBA should work as well.

### Version and OS

 - OpenModelica Version: master branch
 - OS: Ubuntu Linux 64-bit

**Comment by casella** [comment]
@anotheruserofgithub the problem is that currently OMOptim is not maintained, we don't have the resources to do that. If you want to help us fix this kind of issues for fun, you are very welcome to do so, just let us know 😃

**Comment by anotheruserofgithub** [comment]
Actually it was very easy to fix on my own, see PR OpenModelica/OMOptim#18.

**Comment by casella** [comment]
Thanks!
